packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -174,7 +184,7 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
       name: initialValue?.name || "",
       hypothesis: initialValue?.hypothesis || "",
       activationMetric: initialValue?.activationMetric || "",
-      hashVersion: initialValue?.hashVersion || 2,
+      hashVersion: initialValue?.hashVersion || hasSDKWithNoBucketingV2 ? 1 : 2,
```

Comment:
I think this is missing parentheses.
```suggestion
      hashVersion: initialValue?.hashVersion || (hasSDKWithNoBucketingV2 ? 1 : 2),
```